src/services/fileSystem.ts

```diff
@@ -25,19 +27,19 @@ export const deleteStoredWorkpass = () => {
   return FileSystem.deleteAsync(DEFAULT_FILE_NAME);
 };
 
-const TIME_ACCEPTED_FILE_NAME = "@storedTimeAccepted";
-
-export const setTimeAccepted = time => {
-  return FileSystem.writeAsStringAsync(TIME_ACCEPTED_FILE_NAME, time);
+export const storeTime = async () => {
+  await AsyncStorage.setItem("@storedTimeAccepted", getCurrentDateAndTime());
```

Comment:
No need async await.

```suggestion
  return AsyncStorage.setItem("@storedTimeAccepted", getCurrentDateAndTime());
```